modules/javafx.graphics/src/main/java/javafx/scene/image/Image.java

```diff
@@ -741,9 +741,9 @@ public Image(@NamedArg("is") InputStream is, @NamedArg("requestedWidth") double
     }
 
     /**
-     * Package private internal constructor used only by {@link WritableImage}.
+     * Package private internal constructor used only by {@code WritableImage}.
      *
-     * @param pixelBuffer the {@code PixelBuffer} to construct from.
+     * @param pixelBuffer the {@code PixelBuffer} used to construct this image.
```

Comment:
This is internal doc. These changes are to match the documentation with `WritableImage` class.